kolibri/core/auth/api.py

```diff
@@ -259,6 +259,32 @@ def perform_create(self, serializer):
         self.set_password_if_needed(instance, serializer)
 
 
+class ExistingUsernameView(views.APIView):
+    def get(self, request):
+        username = request.GET.get("username")
+        facility_id = request.GET.get("facility")
```

Comment:
This does not change the implementation, it's just that we prefer to use `request.query_params` in the rest of the codebase.

```suggestion
        username = request.query_params.get("username")
        facility_id = request.query_params.get("facility")
```